app/server/src/jsonResponse.ts

```diff
@@ -20,7 +16,7 @@ export const jsonResponseSuccess = (data: any, res: Response) => {
 
 export const jsonResponseError = (
     httpStatus: number,
-    error: ErrorCode,
+    error: string,
```

Comment:
This avoids a circular dependency - it's fine for the response builder to treat error type as a string. 
(ErrorCode enum was moved, and renamed ErrorType to distinguish from the uids we generate.)